src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10469,84 +10469,19 @@ void func_80848EF8(Player* this) {
-    /* 0x07 */ 13,
-    /* 0x08 */ 14,
-    /* 0x09 */ 15,
-    /* 0x0A */ 16,
-    /* 0x0B */ 17,
-    /* 0x0C */ 18,
-    /* 0x0D */ -22,
-    /* 0x0E */ 23,
-    /* 0x0F */ 24,
-    /* 0x10 */ 25,
-    /* 0x11 */ 26,
-    /* 0x12 */ 27,
-    /* 0x13 */ 28,
-    /* 0x14 */ 29,
-    /* 0x15 */ 31,
-    /* 0x16 */ 32,
-    /* 0x17 */ 33,
-    /* 0x18 */ 34,
-    /* 0x19 */ -35,
-    /* 0x1A */ 30,
-    /* 0x1B */ 36,
-    /* 0x1C */ 38,
-    /* 0x1D */ -39,
-    /* 0x1E */ -40,
-    /* 0x1F */ -41,
-    /* 0x20 */ 42,
-    /* 0x21 */ 43,
-    /* 0x22 */ 45,
-    /* 0x23 */ 46,
-    /* 0x24 */ 0,
-    /* 0x25 */ 0,
-    /* 0x26 */ 0,
-    /* 0x27 */ 67,
-    /* 0x28 */ 48,
-    /* 0x29 */ 47,
-    /* 0x2A */ -50,
-    /* 0x2B */ 51,
-    /* 0x2C */ -52,
-    /* 0x2D */ -53,
-    /* 0x2E */ 54,
-    /* 0x2F */ 55,
-    /* 0x30 */ 56,
-    /* 0x31 */ 57,
-    /* 0x32 */ 58,
-    /* 0x33 */ 59,
-    /* 0x34 */ 60,
-    /* 0x35 */ 61,
-    /* 0x36 */ 62,
-    /* 0x37 */ 63,
-    /* 0x38 */ 64,
-    /* 0x39 */ -65,
-    /* 0x3A */ -66,
-    /* 0x3B */ 68,
-    /* 0x3C */ 11,
-    /* 0x3D */ 69,
-    /* 0x3E */ 70,
-    /* 0x3F */ 71,
-    /* 0x40 */ 8,
-    /* 0x41 */ 8,
-    /* 0x42 */ 72,
-    /* 0x43 */ 73,
-    /* 0x44 */ 78,
-    /* 0x45 */ 79,
-    /* 0x46 */ 80,
-    /* 0x47 */ 89,
-    /* 0x48 */ 90,
-    /* 0x49 */ 91,
-    /* 0x4A */ 92,
-    /* 0x4B */ 77,
-    /* 0x4C */ 19,
-    /* 0x4D */ 94,
+    PLAYER_CSMODE_NONE, PLAYER_CSMODE_3,    PLAYER_CSMODE_3,    PLAYER_CSMODE_5,   PLAYER_CSMODE_4,   PLAYER_CSMODE_8,
+    PLAYER_CSMODE_9,    PLAYER_CSMODE_13,   PLAYER_CSMODE_14,   PLAYER_CSMODE_15,  PLAYER_CSMODE_16,  PLAYER_CSMODE_17,
+    PLAYER_CSMODE_18,   -PLAYER_CSMODE_22,  PLAYER_CSMODE_23,   PLAYER_CSMODE_24,  PLAYER_CSMODE_25,  PLAYER_CSMODE_26,
+    PLAYER_CSMODE_27,   PLAYER_CSMODE_28,   PLAYER_CSMODE_29,   PLAYER_CSMODE_31,  PLAYER_CSMODE_32,  PLAYER_CSMODE_33,
+    PLAYER_CSMODE_34,   -PLAYER_CSMODE_35,  PLAYER_CSMODE_30,   PLAYER_CSMODE_36,  PLAYER_CSMODE_38,  -PLAYER_CSMODE_39,
+    -PLAYER_CSMODE_40,  -PLAYER_CSMODE_41,  PLAYER_CSMODE_42,   PLAYER_CSMODE_43,  PLAYER_CSMODE_45,  PLAYER_CSMODE_46,
+    PLAYER_CSMODE_NONE, PLAYER_CSMODE_NONE, PLAYER_CSMODE_NONE, PLAYER_CSMODE_67,  PLAYER_CSMODE_48,  PLAYER_CSMODE_47,
+    -PLAYER_CSMODE_50,  PLAYER_CSMODE_51,   -PLAYER_CSMODE_52,  -PLAYER_CSMODE_53, PLAYER_CSMODE_54,  PLAYER_CSMODE_55,
+    PLAYER_CSMODE_56,   PLAYER_CSMODE_57,   PLAYER_CSMODE_58,   PLAYER_CSMODE_59,  PLAYER_CSMODE_60,  PLAYER_CSMODE_61,
+    PLAYER_CSMODE_62,   PLAYER_CSMODE_63,   PLAYER_CSMODE_64,   -PLAYER_CSMODE_65, -PLAYER_CSMODE_66, PLAYER_CSMODE_68,
+    PLAYER_CSMODE_11,   PLAYER_CSMODE_69,   PLAYER_CSMODE_70,   PLAYER_CSMODE_71,  PLAYER_CSMODE_8,   PLAYER_CSMODE_8,
+    PLAYER_CSMODE_72,   PLAYER_CSMODE_73,   PLAYER_CSMODE_78,   PLAYER_CSMODE_79,  PLAYER_CSMODE_80,  PLAYER_CSMODE_89,
+    PLAYER_CSMODE_90,   PLAYER_CSMODE_91,   PLAYER_CSMODE_92,   PLAYER_CSMODE_77,  PLAYER_CSMODE_19,  PLAYER_CSMODE_94,
```

Comment:
I'm not sure if player cues will eventually get enums or just be left at numbers, but I think it would still be nice to have this data indexed with `/* 0x00 */` or `// 0x00` (i.e. how fast can you tell me which of these is player cue 55)

Reply:
you can hover the enum and get the value pretty easily. for something like this where its not indexed by some other named enum, I personally dont see much value in it

Reply:
It's not a 1-to-1 mapping though, `D_808547C4` maps player cues to player csModes. So playerCue 55 would be `PLAYER_CSMODE_63`

Reply:
oh I did miss that
in that case it should prob be an enum for cue id, which would need to be created. and I think thats pretty out of scope of this pr (though it can be done soon, and I do plan on looking at player cs stuff soon)

Reply:
Cool, so if there's consideration being made for player cue ids, this can be done later then